build/gen_stub.php

```diff
@@ -616,13 +633,16 @@ function parseFunctionLike(
                 $isDeprecated = true;
             } else if ($tag->name === 'return') {
                 $haveDocReturnType = true;
+            } else if ($tag->name === 'param') {
+                $docParamTypes[$tag->getVariableName()] = true;
             }
         }
     }
 
     $args = [];
     $numRequiredArgs = 0;
     $foundVariadic = false;
+    static $warnings = [];
```

Comment:
Hack to only show the warning once for the same function

Reply:
Uh, why do we visit the same function multiple times?

Reply:
No, that's not the case :) by default, we'd report as many errors as parameters without type info we have. And this is what I wanted to avoid. Unfortuantely, I was too much in a hurry, and didn't notice that there's no need for a static variable, a simple `bool` flag would be just enough.